### PR TITLE
fix(chat): Fix primary color selection on quotes

### PR DESCRIPTION
**Pull request by nickvergessen**
### ☑️ Resolves

* Fix my primary color is pink, but the quotes were blue

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
Bildschirmfoto vom 2023-08-25 10-13-02 | Bildschirmfoto vom 2023-08-25 10-13-10

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not required
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: src/components/Quote.vue [hunk 1]
```diff
@@ -283,7 +283,7 @@ export default {
 	}
 
 	&.quote-own-message::before {
-		background-color: var(--color-primary-default);
```
Comment (Antreesy) [inline src/components/Quote.vue:286]: Shouldn't it be aligned with your theming as well?

https://github.com/nextcloud/server/blob/85ff1449d33df76db17d741e46485799e7121977/apps/theming/lib/Themes/CommonThemeTrait.php#L58

**Review by Antreesy — APPROVED** [review]
Fine as a quick fix, but not sure, whether it should be upstreamed or not

File: src/components/Quote.vue (hunk 1 above)
Comment (nickvergessen) [reply src/components/Quote.vue:286]: No, default is default blue
The other option would have been `--color-primary` but that could be white on white, etc. So we have to use element here

**Pull request merged**